docker/install/ubuntu_install_hexagon.sh

```diff
@@ -21,9 +21,26 @@ set -o pipefail
 
 # Install LLVM/clang
 CLANG_LLVM_HOME=/opt/clang-llvm
```

Comment:
`CLANG_LLVM_HOME` is used in hexagon TVM build. Can you change this script to install LLVM in `CLANG_LLVM_HOME` path